src/components/common/buttonCta/buttonCta.tsx

```diff
@@ -0,0 +1,18 @@
+import {ButtonHTMLAttributes, ReactNode} from "react";
+import style from "./buttonCta.module.scss";
+interface Props extends ButtonHTMLAttributes<HTMLInputElement> {
+  text: string;
+}
+
+function ButtonCta({className = "", text, type}: Props) {
+  const classnames = `${style.cta} ${className}`;
+  return (
+    <button className={classnames} type={type}>
+      {text}
```

Comment:
버튼에 들어갈 텍스트는 children으로 받는게 더 좋을 것 같아요~
```suggestion
      {children}
```